seacrowd/sea_datasets/ara_close/ara_close.py

```diff
@@ -0,0 +1,222 @@
+                version=datasets.Version(_SEACROWD_VERSION),
+                description=f"{_DATASETNAME} SEACrowd schema",
+                schema="seacrowd_text",
+                subset_id=f"{_DATASETNAME}",
+            )
+            for lang in _LANGUAGES
+        ]
+    )
+
+    DEFAULT_CONFIG_NAME = f"{_DATASETNAME}_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "title": datasets.Value("string"),
+                    "text": datasets.Value("string"),
+                    "label": datasets.Value("string"),
+                }
+            )
+
+        elif self.config.schema == "seacrowd_text":
+            features = schemas.text_features(["1", "2", "3"])
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+
+        lang = self.config.name.split("_")[2]
+        if lang in _LANGUAGES:
+            data_path = Path(dl_manager.download_and_extract(_URLS[lang]))
+        else:
+            data_path = [Path(dl_manager.download_and_extract(_URLS[lang])) for lang in _LANGUAGES]
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepath": data_path,
+                    "split": "train",
+                },
+            )
+        ]
+
+    def _generate_examples(self, filepath: Path, split: str) -> Tuple[int, Dict]:
+        """Yields examples as (key, example) tuples."""
+        lang = self.config.name.split("_")[2]
+        if lang in _LANGUAGES:
+            file_content = open(filepath, "r").readlines()
+        else:
+            file_content = []
+            for path in filepath:
+                lines = open(path, "r").readlines()
+                file_content.extend(lines)
+
+        if self.config.schema == "source":
+            idx = 0
+            for line in file_content:
+                split_data = line.strip().split(",")
+                title = split_data[0]
+                label = split_data[1]
+                text = ",".join(split_data[2:])
+                ex = {"title": title, "text": text, "label": label}
+                yield idx, ex
+                idx += 1
+
+        elif self.config.schema == "seacrowd_text":
+            idx = 0
+            for line in file_content:
+                split_data = line.strip().split(",")
+                title = split_data[0]
+                label = split_data[1]
+                text = ",".join(split_data[2:])
+                ex = {
+                    "id": title,
+                    "text": text,
```

Comment:
Might need second opinion from another reviewer (cc: @gentaiscool ), should the `id` here be restricted to ordinal numbers? (e.g., 0, 1,2, 3, 4). Is the `title` acceptable here? The test errors out when this runs (IDs aren't globally unique) so I don't think we should use the `title` imo

Reply:
I agreed it has to be ordinal number